components/Introduction.js

```diff
@@ -1,7 +1,37 @@
 import React from "react";
+import { Row, Col, Container } from "react-bootstrap";
 
 const Introduction = () => {
-	return <div>Introduction</div>;
+	return (
+		<Container>
```

Comment:
you can remove the `Container` since there is no styling applied and it is not being used